install/centos-installation-guide.md

```diff
@@ -15,7 +15,8 @@ For more information on installing Docker please refer to the
 
 2. Install the Kata Containers components with the following commands:
 
-  **Note:** The repository redirects the download content to use `http`, be aware that this installation channel is not secure.
+>  **Note:** The repository redirects the download content to use `http`, be aware that this installation channel is not secure.
```

Comment:
As I know it will now bug you.... two spaces here after the `>`, but only one in the previous one ;-)

Reply:
Ta - fixed! ;)